webapp/carts/models.py

```diff
@@ -0,0 +1,42 @@
+from django.conf import settings
+from django.db import models
+
+
+User = settings.AUTH_USER_MODEL
+
+
+class Cart(models.Model):
```

Comment:
Would this model currently be able to force the system to only have one active cart for a user at a given time?
How is this case handled?

Reply:
there is going to be a cart manger that will handle both authenticated and guest checkout essentially we need the ability to pull the users info for easy checkout as well as do some session management.

Reply:
I'm not sure how that addresses the question 🙁 
I mean, how is the DB to natively prevent multiple carts being associated to a user.
What benefit does a Manager provide that can't be achieved with explicit DB requirements?